src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -73,6 +73,7 @@ SolverBase::SolverBase( std::string const & name,
   registerWrapper( viewKeyStruct::initialDtString, &m_nextDt )->
```

Comment:
@cssherman Any objections giving `m_nextDt` it's own string, creating a `m_initialDtString` to hold the value. We can then just set the value of `m_nextDt` in one of the post processing routines.

Reply:
Nope, it just needs to be present somewhere, so that the first dt after restarting is calculated correctly